pkg/sysconfwatch/sysconfwatch_linux.go

```diff
@@ -91,6 +91,11 @@ func (c *SysConfWatchController) Run(ctx context.Context, ready chan<- struct{},
 	var buf []byte = make([]byte, 8)
 	// Take a snapshot of the system and monototic clocks as a base reference
 	stimeRef, mtimeRef := getSysMonTimes()
+
+	go func() {
+		klog.Infof("sysconfwatch-controller is ready")
+		close(ready)
+	}()
```

Comment:
Can we do the same without a new goroutine?

```suggestion
	klog.Infof("sysconfwatch-controller is ready")
	close(ready)
```